service-deployment/clusterconfig-example.yaml

```diff
@@ -81,8 +81,10 @@ clusterinfo:
   webportalinfo:
     # uri for rest server
     rest_server_uri: http://10.0.3.9:9186
-    # uri for cluster status dashboard
-    cluster_status_uri: http://10.0.3.9:9090
+    # uri of k8s dashboard
+    k8s_dashboard_uri: http://10.0.3.9:9090
+    # uri of grafana portal
+    grafana_uri: http://10.0.3.9:9090
```

Comment:
>http://10.0.3.9:9090 [](start = 17, length = 20)

this reminds me that eventually we need a tool to check if the config is wrong. k8s dashboard and grafana has the same uri due to misconfig could indeed happen.

Reply:
That's right. The tool is to do a semantic check over deployment settings, making the deployment follow some predefined policies. Shall we put this one into our backlog?

For the current code: These urls are placeholders. Specific values vary according to actual deployment.

Reply:
yes. it is a good idea to put this in backlog 

---
In reply to: [158664762](https://github.com/Microsoft/pai/pull/136#discussion_r158664762) [](ancestors = 158664762)